modules/nw-openstack-external-ccm.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// tbd
+// TODO: GitHub link clearance.
+
+:_content-type: CONCEPT
+[id="nw-openstack-external-ccm_{context}"]
+= The {rh-openstack} Cloud Controller Manager (CCM)
+
+In {product-title} 4.12, clusters that run on {rh-openstack-first} are switched from the legacy {rh-openstack} cloud provider to the external link:https://github.com/kubernetes/cloud-provider-openstack[OpenStack CCM]. This change follows the move in Kubernetes from in-tree, legacy cloud providers to external cloud providers that are implemented by using the link:https://kubernetes.io/docs/concepts/architecture/cloud-controller/[Cloud Controller Manager].
+
+To preserve user-defined configurations for the legacy cloud provider, existing configurations are mapped to new ones as part of the migration process. It searches for a configuration called `cloud-provider-config` in the `openshift-config` namespace.
+
+[NOTE]
+====
+The config map name `cloud-provider-config` is not statically configured. It is derived from the `spec.cloudConfig.name` value in the `infrastructure/cluster` CRD.
+====
+
+Found configurations are synchronized to the `cloud-conf` config map in the `openshift-cloud-controller-manager` namespace.
+//QE: Verify
+To synchronize the configuration to a different namespace, you can override the default `openshift-cloud-controller-manager` namespace by passing the name of the namespace with the `--namespace` flag to the operator binary.
```

Comment:
I believe the operator will enforce the openshift-cloud-controller-manager since the manifest does not allow setting that, so perhaps is better not to mention.

Reply:
```suggestion
// To synchronize the configuration to a different namespace, you can override the default `openshift-cloud-controller-manager` namespace by passing the name of the namespace with the `--namespace` flag to the operator binary.
```